data/migrations/20200117135320_07-project-requirements.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable func-names */
+exports.up = function(knex) {
+  return knex.schema.createTable('project_requirements', table => {
+    table.increments();
+    table
+      .integer('event_id')
+      .unsigned()
+      .notNullable();
+    table
+      .foreign('event_id')
+      .references('events.id')
+      .onUpdate('CASCADE')
+      .onDelete('CASCADE');
+    table.boolean('video_url');
+    table.boolean('git_url');
```

Comment:
Give this boolean columns a default value of false.
e.g
`table.boolean('video_url').defaultTo(false)`

Reply:
done